threeD-components/Inventory.js

```diff
@@ -183,7 +184,8 @@ export const Inventory = (props = {}) => {
       }
       .details .button {
         border: 5px solid;
-        padding: 10px;
+        padding: 20px;
+        margin: 20px 0px 20px 0px;
```

Comment:
Prefer this to be explicit direction overrides, not a circle ordering.

Reply:
git gud

Reply:
I don't understand.